ext/hash/hash.c

```diff
@@ -1392,7 +1406,7 @@ static zend_object *php_hashcontext_clone(zend_object *zobj) {
 	newobj->ops = oldobj->ops;
 	newobj->options = oldobj->options;
 	newobj->context = php_hash_alloc_context(newobj->ops);
-	newobj->ops->hash_init(newobj->context);
+	newobj->ops->hash_init(newobj->context, oldobj->args);
```

Comment:
Suggest passing `NULL` here: the upcoming `hash_copy` should do the work necessary for initializing the context from `oldobj`, including any initialization steps implied by `oldobj->args`.